notebooks/cleaning_data.ipynb

```diff
@@ -0,0 +1,2892 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    for prefix in prefixes:
</pre><p>Using the&nbsp;<em>display_indirect_missing&nbsp;</em>function from the&nbsp;<em>CLD&nbsp;</em>utility file as cell 13 will let you to see the percentages of the indirect missing data.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Clustering-Analysis/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>